src/data/formatDate.test.js

```diff
@@ -0,0 +1,8 @@
+import formatDate from './formatDate';
+
+test('16:11:2021 should be returnded', () => {
```

Comment:
it is illogical. message should looks like this - `methodName` should return `result` if  (probably here can be condition).
for example: "formatDate should return `16:11:2021` if input is correct date". Test should have at least 2 test-cases: successful and unsuccessful.

Reply:
I correct tests and add a few new.
```suggestion
import { formatDate, getYears }from './formatDate';

test('Format date should be return 2021-11-16', () => {
  const input = '1637060837011';
  const actual = formatDate(input);
  const expected = '2021-11-16';
  expect(actual).toBe(expected);
}, 0);

test('Format date should be return 2021-11-16', () => {
  const input = '2021-11-16';
  const actual = formatDate(input);
  const expected = '2021-11-16';
  expect(actual).toBe(expected);
}, 0);


test('Get years should be return 17', () => {
  const input = '1072990800000';
  const actual = getYears(input);
  const expected = 18;
  expect(actual).toBe(expected);
}, 0);

test('Get years should be return 0', () => {
  const input = '01-20-2022';
  const actual = formatDate(input);
  const expected = 0;
  expect(actual).toBe(expected);
}, 0);
```